pkg/client/mock/client_generated.go

```diff
@@ -38,6 +37,7 @@ func (m *MockClient) EXPECT() *MockClientMockRecorder {
 
 // DescribeImages mocks base method
 func (m *MockClient) DescribeImages(arg0 *ec2.DescribeImagesInput) (*ec2.DescribeImagesOutput, error) {
+	m.ctrl.T.Helper()
```

Comment:
I'm unsure why these are added.  I think this file hadn't been generated in quite some time.  I utilized the mockgen in ./hack/tools/bin to generate this file, FYI.  A newer version of mockgen I have installed locally yielded the same results.